### ISSUE TITLE
rofi-pass isn't passing the `-config` option to rofi

### ISSUE THREAD

**Issue by Rahtgaz** [carnager/rofi-pass#89]
Changing the rofi command in `rofi-pass.conf` to include a rofi config file is not working:

```
# ~/.config/rofi-pass/config

# rofi command. Make sure to have "$@" as last argument
_rofi () {
    rofi -i -width 31 -no-auto-select -config $HOME/.config/i3/rofi.conf "$@"
}
```

This would be greatly useful when we wish to style rofi-pass differently from rofi, or when our rofi `.conf` is not in a standard location.

**Comment by Narrat** [comment]
Are you using latest release or a snapshot of master? There was #87 which was about not applied rofi options

**Comment by Rahtgaz** [comment]
Well, I don't definitely have that fix. I'll fetch from origin immediately. Thanks.

**Comment by carnager** [comment]
Just released 1.5.2 with this fix.
